src/index.html

```diff
@@ -1,13 +1,233 @@
+          <img class="explore-online__photo" src="images/image8-square.png" alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Collection Areas</h3>
+          <p>These collection areas study, exhibit, and care for the objects in The Met collection.</p>
+        </div>
+      </div>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img class="explore-online__photo" src="images/image9-square.png" alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Travel</h3>
+          <p>Travel with The Met connects the Museum's wide and diverse audience to art, inspiration, and historical perspective.</p>
+        </div>
+      </div>
+      <div class="explore-online__photos">
+        <a href="#" class="explore-online__link">
+          <img class="explore-online__photo" src="./images/image10-square.png" alt="image">
+        </a>
+        <div class="explore-online__text">
+          <h3>Virtual group tours</h3>
+          <p> Request a guided tour, reserve admission and lead your own small group, or request a live virtual tour led by a Museum guide.</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="contacts" id="contacts">
+      <h2>Contact us</h2>
+      <div class="contacts__container__wrapper">
+        <div class="contacts__container">
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Fifth Avenue</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a href="tel:+1 212-535--7710" class="contacts__title--content">+1 212-535--7710</a>
+              <p class="contacts__title--title">Address</p>
+              <a href="https://maps.app.goo.gl/zW1hK1uyAj9jKZiR9" target="_blank">1000 Fifth Avenue <br>
+              New York, NY 10028</a>
+            </div>
+          </div>
+          <div class="contacts__container--card">
+            <p class="contacts__title">The Met Cloisters</p>
+            <div class="contacts__wrapper">
+              <p class="contacts__title--title">Phone</p>
+              <a href="tel:+1 212-923-3700" class="contacts__title--content">+1 212-923-3700</a>
+              <p class="contacts__title--title">Address</p>
+              <a href="https://maps.app.goo.gl/Dd9D2kouF1f8HZ5r9" target="_blank">99 Margaret Corbin Drive <br>
+                Fort Tryon Park<br>New York, NY 10040</a>
+            </div>
+          </div>
+      </div>
+      <div class="contacts__container__wrapper">
+        <form onsubmit="this.reset(); return false;">
+          <input placeholder="Name" required>
+          <input placeholder="Email" required>
+          <textarea placeholder="Message" required></textarea>
+          <button class="button">Send</button>
+        </form>
+      </div>
+    </section>
+      <footer class="footer">
```

Comment:
You should make the footer sticky [Checklist #16](https://github.com/mate-academy/layout_landing-page/blob/main/checklist.md#:~:text=Make%20the%20footer%20sticky%20at%20the%20bottom%20with%20z%2Dindex%3A%20%2D1%20to%20get%20a%20fixed%20background%20effect.)